packages/react-charting/src/components/VerticalBarChart/VerticalBarChart.base.tsx

```diff
@@ -318,10 +342,14 @@ export class VerticalBarChartBase extends React.Component<IVerticalBarChartProps
     containerWidth: number,
     xElement?: SVGElement | null,
   ) => {
-    return (this._bars =
-      this._xAxisType === XAxisTypes.NumericAxis
-        ? this._createNumericBars(containerHeight, containerWidth, xElement!)
-        : this._createStringBars(containerHeight, containerWidth, xElement!));
+    if (!this._isChartEmpty) {
```

Comment:
why is any of this change needed.

Reply:
removed after conditionally rendering the empty aria label div.